File: fibonacci.R
fib <- function(nterms) {
  # check if number is great than 0
  if (nterms <= 0) {
    print("Choose a larger number")
  } else if (nterms ==1){
    # check if number is equal to 1
    return(1)
  } else if (nterms ==2){
    # check if number is equal to 2
    return(1)
  } else {
    return(fib(nterms-1)+fib(nterms-2))
  }
 
}

File: fibonacci.py
#!/usr/bin/env python

def fib(nterms):
    # check if nterms is lower than zero
    if nterms <= 0:
        print('choose a larger number')
        return
    # check if neterms is 1
    elif nterms==1: 
        result=1
    # check if nterms is 2
    elif nterms==2:
        result=1
    else:
        result= fib(nterms-1)+fib(nterms-2)
    return result

for i in range(1,15):
    print(fib(i))
